llm/llama.go

```diff
@@ -471,6 +471,9 @@ func (llm *llama) Embedding(input string) ([]float64, error) {
 		return nil, errors.New("llama: eval")
 	}
 
+	timings := C.llama_get_timings(llm.ctx)
+	log.Printf("embed: eval_ms=%v", parseDurationMs(float64(timings.t_p_eval_ms)))
```

Comment:
You should use `C.llama_print_timings` instead

Reply:
even better, thanks